Civi/CCase/SequenceListener.php

```diff
@@ -48,7 +48,7 @@ public function onCaseChange(\Civi\CCase\Event\CaseChangeEvent $event) {
     }
 
     $actTypes = array_flip(\CRM_Core_PseudoConstant::activityType(TRUE, TRUE, FALSE, 'name'));
-    $actStatuses = array_flip(\CRM_Core_PseudoConstant::activityStatus('name'));
+    $actStatuses = array_flip(\CRM_Activity_BAO_Activity::getStatusesByType(\CRM_Activity_BAO_Activity::COMPLETED));
```

Comment:
What does this function do and does it also need to respond to CANCELLED activities?

Reply:
This function calls the next activity in sequence when the current activity is set to `Completed`. It was only checking for the single activity status before(Completed - line 62 below). Now, after the above changes, it triggers the creation of next activity when the current activity is updated to any of the statuses of TYPE = `COMPLETED`.

I'm not really sure if it makes sense to create next activity when the present activity is CANCELLED. But, this was not the case before this PR.

Reply:
Ok, can you add what you just said about what the function does into the docblock for future reference?

Reply:
Done 👍